coprocessor/fhevm-engine/transaction-sender/Cargo.toml

```diff
@@ -33,6 +33,7 @@ humantime = { workspace = true }
 
 # local dependencies
 fhevm-engine-common = { path = "../fhevm-engine-common" }
+fhevm_gateway_bindings = { path = "../../../gateway-contracts/rust_bindings" }
```

Comment:
Just for the record, remember to replace this with git deps here and at the gw-listener component. Also, to rollback changes in the make command at the Gateway for generating the bindings. cc @eudelins-zama